src/core/github.js

```diff
@@ -21,9 +21,9 @@ export const github = new Promise((resolve, reject) => {
 
 const localizationStrings = {
   en: {
-    file_a_bug: "File a bug",
-    participate: "Participate",
-    commit_history: "Commit history",
+    file_a_bug: "File a bug:",
+    participate: "Participate:",
+    commit_history: "Commit history:",
   },
   ko: {
     participate: "참여",
```

Comment:
Shouldn't these also be updated? And should there be `colon` field instead of repeating it everywhere? 🤔

Reply:
Ah, sorry. I wasn't sure if ko also used ":". I'll update those.